packages/i18n/src/shared/en/prompts.json

```diff
@@ -121,6 +121,7 @@
     "root": "all food categories",
     "back": "Back to '{category}'",
     "none": "No food results. Please try refining your search.",
+    "refine": "Our search has found a lot of results. Try refining your search term.",
```

Comment:
It might be an issue for other languages that exist in the codebase if this field is only presented in the `en` file.  A straight copy (without translation) would be fine.

Reply:
Thanks a lot, Dinislam. Will change it to a straight copy.

Reply:
Hi Dinislam. Lukas suggests using i18n, but creating the key value pair for each language file.